tests/framework/validators/ImageValidatorTest.php

```
@@ -0,0 +1,155 @@
+            'name' => $fileName,
+            'tempName' => $filePath,
+            'type' => 'image/jpeg',
+            'size' => filesize($filePath),
+            'error' => UPLOAD_ERR_OK,
+        ]);
+    }
+}
+
+class ImageValidatorViewStub extends View
+{
+    public function registerAssetBundle($name, $position = null)
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$name'. (undefined)

(UnusedFormalParameter)